lbry/tests/unit/stream/test_stream_descriptor.py

```diff
@@ -78,6 +78,11 @@ def _write_sd(self):
         self.sd_dict['blobs'][-2]['length'] = 0
         await self._test_invalid_sd()
 
+    def test_sanitize_file_name(self):
+        test_cases = [' t/-?t|.g.ext ', 'end_dot .', '.file\0\0', 'test n\16ame.ext', 'COM8', 'LPT2', '']
+        expected = ['t-t.g.ext', 'end_dot', '.file', 'test name.ext', '', '', '']
+        actual = [sanitize_file_name(tc) for tc in test_cases]
+        self.assertListEqual(actual, expected)
```

Comment:
It's difficult to read and debug tests written like this. Can you spell out each test on its own line?

Example from other tests:
https://github.com/lbryio/lbry-sdk/blob/9bb0ca2f0d7ef428d17ee8bdec89d0bb17f99dca/lbry/tests/unit/schema/test_tags.py#L12-L16

https://github.com/lbryio/lbry-sdk/blob/9bb0ca2f0d7ef428d17ee8bdec89d0bb17f99dca/torba/tests/client_tests/unit/test_utils.py#L10-L13